acceptance-tests/src/test/java/tech/pegasys/eth2signer/dsl/lotus/FilecoinJsonRequests.java

```diff
@@ -97,11 +99,27 @@ public static Boolean walletVerify(
         .execute();
   }
 
+  public static FilecoinSignedMessage walletSignMessage(
+      final JsonRpcClient jsonRpcClient,
+      final String address,
+      final Map<String, Object> messageMap) {
+    return jsonRpcClient
+        .createRequest()
+        .method("Filecoin.WalletSignMessage")
+        .id(101)
+        .params(address, messageMap)
+        .returnAs(FilecoinSignedMessage.class)
+        .execute();
+  }
+
   public static String executeRawJsonRpcRequest(final String url, final String request)
       throws IOException {
     final HttpPost post = new HttpPost(url);
     post.setEntity(new StringEntity(request, Charsets.UTF_8));
     post.setHeader(HttpHeaders.CONTENT_TYPE, MediaType.JSON_UTF_8.toString());
+    post.setHeader(
```

Comment:
this can go away (or can be conditionally added via env variable).

Reply:
done